packages/lesswrong/components/ea-forum/TargetedJobAdSection.tsx

```diff
@@ -202,7 +202,7 @@ const TargetedJobAdSection = () => {
         jobName: activeJob,
         adState: 'seen'
       }
-    })
+    }).finally(refetchUserJobAds)
```

Comment:
@Will-Howard Ok I looked into this a bit more. I was able to get into a bad state by juggling two separate tabs such that they both tried to create a new record at approximately the same time. Whether the create succeeds or fails (like when the user+job pair already exists), this seems like a reasonable time to refetch the list, so I added this here.

I think this could basically fix the issue even if I haven't discovered the exact problem yet. Let me know if you think this is reasonable. If so then I'll go ahead and deploy this.

Reply:
👍  I think this is reasonable, I haven't independently verified it though